draft-ietf-tls-tls13.md

```diff
@@ -4695,22 +4695,34 @@ concerned with:
   handshake with B and probably retry the request, leading to duplication on
   the server system as a whole.
 
-The first class of attack can be prevented by the mechanism described
-in this section.  Servers need not permit 0-RTT at all, but those
-which do SHOULD implement either the single-use tickets or
-ClientHello recording techniques described in the following two
-sections.
+The first class of attack can be prevented by sharing state to provide a
+guarantee that the 0-RTT is only accepted once.  Servers SHOULD provide that
```

Comment:
s/to provide a guarantee/to guarantee/

Reply:
s/that the 0-RTT/that 0-RTT data/

Reply:
We've been careful to say "at most once" elsewhere, which is more correct.

Reply:
I would prefer "that the 0-RTT data", using the definitive article to bind the text to a specific instance of 0-RTT data.

Reply:
There is no specific instance of 0-RTT data here.

Reply:
Meh. It can go either way, I suppose.